MinimedKit/Models/PumpModel.swift

```diff
@@ -99,6 +99,29 @@ public enum PumpModel: String {
     var usesTwoBytesForMaxBolus: Bool {
         return generation >= 23
     }
+
+    public var supportedBasalRates: [Double] {
+        if generation >= 23 {
+            // 0.025 units (for rates between 0.0-0.975 U/h)
+            let rateGroup1 = ((0...38).map { Double($0) / Double(pulsesPerUnit) })
```

Comment:
Should `38` be `39`?

Reply:
@ps2 Here.

Reply:
Thanks.  Fixed here: https://github.com/ps2/rileylink_ios/commit/23af71639041ca3a2a1b4120d3cd0a06c5d3fa01